osf/models/user.py

```diff
@@ -479,11 +479,6 @@ def csl_given_name(self):
         return utils.generate_csl_given_name(self.given_name, self.middle_names, self.suffix)
 
     def csl_name(self, node_id=None):
-        # disabled users are set to is_registered = False but have a fullname
-        if self.is_registered or self.is_disabled:
-            name = self.fullname
-        else:
-            name = self.get_unclaimed_record(node_id)['name']
```

Comment:
Pretty sure this section should not go away.

Names used on nodes for unclaimed records are local to the node itself. The fullname is just the first name the unclaimed user was given and should not be used everywhere. Instead the unclaimed record on that node should be used as the name for that user. 

Example:
1. My email is added as a new unclaimed user as "Dr. Alexander Joseph Schiller" on Project A.  My fullname is now that. 
2. My email is added as a new unclaimed user as "Alex Schiller" on Project B. With this change, my name would be displayed as "Dr. Alexander Joseph Schiller" in the citations, despite the unclaimed name on this project being "Alex Schiller"

Happy to help if that isn't clear.